lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_clickhouse.conf

```diff
@@ -0,0 +1,95 @@
+emqx_ee_bridge_clickhouse {
+
+    local_topic {
+        desc {
+            en: """The MQTT topic filter to be forwarded to Clickhouse. All MQTT 'PUBLISH' messages with the topic
+matching the local_topic will be forwarded.</br>
+NOTE: if this bridge is used as the action of a rule (EMQX rule engine), and also local_topic is
+configured, then both the data got from the rule and the MQTT messages that match local_topic
+will be forwarded.
+"""
+            zh: """发送到 'local_topic' 的消息都会转发到 Clickhouse。 </br>
+注意：如果这个 Bridge 被用作规则（EMQX 规则引擎）的输出，同时也配置了 'local_topic' ，那么这两部分的消息都会被转发。
+"""
+        }
+        label {
+                en: "Local Topic"
+                zh: "本地 Topic"
+            }
+    }
+    sql_template {
+        desc {
+            en: """SQL Template. The template string can contain placeholders
+for message metadata and payload field. The placeholders are inserted
+without any checking and special formatting so it is important to
+ensure that the inserted values are formatted and escaped correctly."""
+            zh: """"""
+            }
+        label {
+            en: "SQL Template"
+            zh: "SQL 模板"
+        }
+    }
+    batch_value_separator {
+        desc {
+            en: """The bridge repeats what comes after the VALUES or FORMAT FormatType in the
+SQL template to form a batch request. The value specified with
+this parameter will be inserted between the values. The default
+value ',' works for the VALUES format but other values
+might be needed if you specify some other format with the 
+clickhouse FORMAT syntax.
+
+See https://clickhouse.com/docs/en/sql-reference/statements/insert-into/ and
+https://clickhouse.com/docs/en/interfaces/formats#formats for more information about
+the format syntax and the available formats."""
+            zh: """"""
```

Comment:
Just a reminder to fill these.

Reply:
Fixed with auto translate. It should be checked by a Chinese speaker @JimMoen